templates/argo-tasks/generate-path.yml

```
@@ -33,6 +36,7 @@ spec:
           - 'generate-path'
           - '--target-bucket-name'
           - '{{=sprig.trim(inputs.parameters.target_bucket_name)}}'
+          - '--no_date_in_survey_path={{=sprig.trim(inputs.parameters.no_date_in_survey_path)}}'
```

I think the parameter needs to be hyphenated:
```suggestion
          - '--no-date-in-survey-path={{=sprig.trim(inputs.parameters.no_date_in_survey_path)}}'
```